src/Pact/Analyze/Types/Shared.hs

```diff
@@ -482,6 +482,23 @@ newtype VarId
   = VarId Int
   deriving (Show, Eq, Enum, Num, Ord)
 
+newtype Munged
+  = Munged Text
+  deriving (Eq, Show)
+
+newtype Unmunged
```

Comment:
Can we get a comment explaining `Munged` / `Unmunged`?

Reply:
Good call. Will do